Source/WebInspectorUI/UserInterface/Views/InlineSwatch.css

```diff
@@ -29,11 +29,13 @@
     position: relative;
     width: 1em;
     height: 1em;
-    margin-right: 3px;
+    margin-right: var(--scope-bar-margin-right-override, var(--scope-bar-margin-right-default));
```

Comment:
this should probably be `--inline-swatch-margin-right-*`

i think that might've been my mistake from when we were chatting offline, so my apologies 😅